glib/src/lib.rs

```diff
@@ -158,78 +158,14 @@ pub use self::source_futures::*;
 mod thread_pool;
 pub use self::thread_pool::ThreadPool;
 
+pub mod thread_guard;
```

Comment:
Any reason for exposing the whole module instead of just the `ThreadGuard`/  `thread_id()` ?

Reply:
I didn't want to pollute the main crate namespace with things that you rarely need, just like `glib::translate`